info/guide/guide-chapter-1.md

```diff
@@ -84,6 +84,8 @@ Package names are all written in lowercase, and consecutive words are concatenat
 package your.company.domain.mobilecontrol.views
 ```
 
+**Note** If there isn't domain name in config file, then package verification will not be carried out.
```

Comment:
I believe we are avoiding technical detail of diktat in code style. It's better to describe it in readme or in comments in yml files.